examples/raw-html/auth-and-singin.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+
+    <!-- Add this to the <head> section of your page -->
+    <!-- OreId -->
+    <!-- TODO: update src -->
+    <script src="https://storage.googleapis.com/oreid-dev-dist/oreid-js/v4/oreid-js.js"></script>
+    <!-- OreId WebWidget -->
+    <!-- TODO: update src -->
+    <script src="https://storage.googleapis.com/oreid-dev-dist/oreid-webpopup/v2/index.js"></script>
```

Comment:
We should use the service urls:
https://service.oreid.io/dist/oreid-js/v4/index.js
https://service.oreid.io/dist/oreid-webpopup/v2/index.js

(these don't work yet because it hasn't been deployed to prod)

Reply:
@yardz Can you look at Jeremy's comment?

Reply:
Done